test/modellib/ConvModel.cpp

```diff
@@ -37,8 +38,8 @@ Conv2DLibBuilder::Conv2DLibBuilder(const std::string &modelName, const int N,
       autoPad(autoPad), pHBegin(pHBegin), pHEnd(pHEnd), pWBegin(pWBegin),
       pWEnd(pWEnd), stride(stride), dilation(dilation), isDynamic(isDynamic) {}
 
-const string Conv2DLibBuilder::getAutoPadName(const ConvAutoPad autoPad) {
-  static const string autoPadName[] = {
+const std::string Conv2DLibBuilder::getAutoPadName(const ConvAutoPad autoPad) {
```

Comment:
Again, just trying to understand the style here. Did it not compile without the `std` qualifier?